### PR TITLE
data: Create sharatkatwa.json

### PR DESCRIPTION
**Pull request by sharatkatwa**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2484"></a>

### CODE REVIEW

File: data/sharatkatwa.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/sharat-katwa",
+      "icon": "FaLinkedin",
+      "color": "#0075D6"
+    }
+  ],
+
+  "milestones": [
+    {
+      "title": "Started Learning about Backend Developer",
```
Comment (iamr-kumar) [inline data/sharatkatwa.json:36]: Maybe you should rephrase it to "Backend Development"?

**Review by eddiejaoude — APPROVED** [review]
Looks good 👍
